deploy/examples/keycloak/rhsso.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
+metadata:
+  name: example-keycloak
+spec:
+  instances: 1
+  externalAccess:
+    enabled: True
+  Profile: RHSSO
```

```suggestion
  profile: RHSSO
```
The p needs to be lower case. @slaskawi

---

Fixed, thanks!